modules/dynamodb/main.tf

```diff
@@ -74,11 +77,12 @@ resource "aws_appautoscaling_target" "dynamodb_table_write_target" {
 }
 
 resource "aws_appautoscaling_policy" "dynamodb_table_read_policy" {
-  name               = "DynamoDBReadCapacityUtilization:${aws_appautoscaling_target.dynamodb_table_read_target.resource_id}"
+  count              = var.billing_mode == "PROVISIONED" ? 1 : 0
```

Comment:
Hadde det gått an å trekke ut `var.billing_mode == "PROVISIONED" ? 1 : 0` til en boolsk variabel? Er det ev. en god ide? Blir potensielt litt mindre sjanse for feil i hvert fall?

Reply:
Det er mulig og en god ide!